src/main/java/org/hydev/mcpm/client/DatabaseManager.java

```diff
@@ -1,35 +1,37 @@
 package org.hydev.mcpm.client;
 
+import org.hydev.mcpm.client.database.LocalPluginTracker;
 import org.hydev.mcpm.client.database.PluginTracker;
 import org.hydev.mcpm.client.database.boundary.SearchPackagesBoundary;
+import org.hydev.mcpm.client.database.fetcher.LocalDatabaseFetcher;
 import org.hydev.mcpm.client.database.inputs.SearchPackagesInput;
 import org.hydev.mcpm.client.database.results.SearchPackagesResult;
+import org.hydev.mcpm.client.database.searchusecase.SearchInteractor;
 import org.hydev.mcpm.client.installer.InstallResult;
 import org.hydev.mcpm.client.installer.input.InstallInput;
 import org.hydev.mcpm.client.models.PluginVersion;
 import org.hydev.mcpm.client.models.PluginYml;
 
+import java.net.URI;
 import java.util.List;
 
 /**
  * Database API
```

Comment:
if you could mention a bit more about what the Database API does that would be great